system/include/emscripten/val.h

```diff
@@ -230,9 +230,6 @@ namespace emscripten {
             #else
             cursor->p = wt;
             // FIXME: This requires the JS reading code to be audited to be compatible with it.
-            assert(false);
-            abort();
-            #endif
```

Comment:
This looks odd.. since you remove the `#endif` how can it compile?   Is the FIXME fixed now?